.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Put a short concise description for your work here.
+
+## Milestone requirements
+
+- [x] task 1
+- [x] task 2
+
+## General requirements
+
+- [x] There are no linter errors.
+- [x] Followed the **Gitflow**.
+- [x] The documentation is professional.
+
+## JavaScript requirements
+
+- [x] Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
+
+## Additional notes
+
+(optional) Please put any additional notes you have here.
+
+Thank you for taking the time to review this pull request.
```

Comment:
[OPTIONAL] It would be nicer to remove the unused file (pull-request-template) or ignore it by git to keep your repository clean.

Reply:
Hi @medaminedev66 

Thank you for the feedback :+1: 
The pull_request_template is actually used to automatically populate the pull request wi a template.
It's part of my CI/CD pipeline. 

Best,

Reply:
That's great 😃